NuGet.config

```diff
@@ -16,7 +16,9 @@
         <add key="repositoryPath" value="..\packages" />
     </config>
   <packageSources>
+    <add key="LocallyBuiltApplicationInsightsPackages" value="..\bin\Debug\NuGet" />
```

Comment:
Yes. I think the project folders may need to be in the same parent folder, but I'm not positive about that.